russh/src/server/encrypted.rs

```diff
@@ -1026,6 +1026,41 @@ impl Session {
                         }
                         Ok(())
                     }
+                    b"streamlocal-forward@openssh.com" => {
+                        let server_socket_path =
+                            std::str::from_utf8(r.read_string().map_err(crate::Error::from)?)
+                                .map_err(crate::Error::from)?;
+                        debug!("handler.streamlocal_forward {:?}", server_socket_path);
+                        let mut client_socket_path = String::from(server_socket_path).clone();
```

Comment:
client_socket_path is a duplicate of server_socket_path here

Reply:
Isn't the point of this so that the server can decide which client_socket_path to return? (based on a handler method accepting server_socket_path)

Reply:
No, there is no second path anywhere in the protocol